src/route/index.js

```diff
@@ -2,6 +2,7 @@ import { createWebHistory, createRouter } from 'vue-router';
 
 import AppLogin from '../views/AppLogin.vue';
 import AppError from '../views/AppError.vue';
+import Dashboard from '../views/dashboard/Dashboard.vue';
```

Comment:
Component names should always be multi-word : https://v3.vuejs.org/style-guide/#multi-word-component-names-essential